src/style.css

```diff
@@ -1 +1,42 @@
-/* add styles here */
+.page {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+.stars {
+  display: flex;
+  width: fit-content;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  background-position: center;
+  margin-right: 4px;
+}
```

Comment:
Consider adding a fallback font in case the main one doesn't load. For example, if you're using a web font for the `.stars__star`, you can specify a generic font family as a backup.